opendatahub-io/opendatahub.io
# Updated Ruby, Updated local testing instructions

Resolves: #73

## Description
Verified and modified the local testing instructions.
Added a troubleshooting section to avoid the issues I ran into while testing (it could be made smaller if those are too obvious)

I have not tested testing using a container because the jekyll testing instructions include instructions on live-reloading the page and require less effort for whoever might want to test the page. (Let me know if I should add detailed instructions for container deployment)

I also updated the Ruby version, and added the webrick library to avoid possible issues when testing.

## How Has This Been Tested?
1. Install all prerequisites.
2. Follow README instructions.
3. Verify changes are applied on the page.

## Merge criteria:
